core/src/main/scala/com/mesosphere/usi/core/Scheduler.scala

```diff
@@ -57,6 +59,137 @@ object Scheduler {
   def fromSnapshot(specsSnapshot: SpecsSnapshot, client: MesosClient): Flow[SpecUpdated, StateOutput, NotUsed] =
     Flow[SpecUpdated].prefixAndTail(0).map { case (_, rest) => specsSnapshot -> rest }.via(fromClient(client))
 
+  def asFlow(specsSnapshot: SpecsSnapshot, client: MesosClient)(
```

Comment:
I wonder if `asFlow` and `asSourceAndSink` are enough and `fromSnapshot` should go. I don't know who would use that.